share/mdnjs/parse.py

```diff
@@ -214,6 +216,18 @@ def writerows(self):
           'disambiguation': disambig
         })
       for mdn in self.inverted_index[keyword]:
+        # add redirect for Web/Api pages
+        if any( word in mdn.title for word in self.CLASS_WORDS ) and len(mdn.title.split('.')) > 1:
```

Comment:
Sorry for not being straingt. I think that `len(mdn.title.split('.')) > 1` can be changed to `'.' in mdn.title`.  I think they both achieve the same result.